app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -78,29 +81,39 @@ fun TokenCategoryDetailScreen(tokenCategory: TokenCategory<*>, onSubcategoryClic
                     }
                 }
             } else {
-                items(tokenCategory.properties) { tokenProperty ->
-                    Spacer(modifier = Modifier.height(OudsSpaceKeyToken.Fixed.Medium.value))
-
-                    tokenProperty.nameRes?.let {
-                        Text(
+                tokenCategory.properties.forEach { tokenProperty ->
+                    item {
+                        Spacer(modifier = Modifier.height(OudsSpaceKeyToken.Fixed.Medium.value))
+                    }
+                    stickyHeader {
+                        tokenProperty.nameRes?.let {
+                            Row(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .background(color = OudsColorKeyToken.Background.Primary.value)
+                            ) {
+                                Text(
+                                    modifier = Modifier
+                                        .fillMaxWidth()
+                                        .padding(all = OudsSpaceKeyToken.Fixed.Medium.value),
```

Comment:
We can simply add the background color before applying the padding to avoid using the `Row`.